src/styles/index.scss

```diff
@@ -168,6 +168,14 @@
     line-height: 20px;
   }
 
+  .font-mega-input {
```

Comment:
can you do this with Tailwind? Also whenever you can, avoid using `px`. Tailwind will by default use `rem` units which are going to make the site look more consistent across different screens.